root/defaults/nginx/site-confs/default.conf.sample

```diff
@@ -68,6 +55,6 @@ server {
 }
 
 # enable subdomain method reverse proxy confs
-include /config/nginx/proxy-confs/*.subdomain.conf;
+include /config/nginx/subdomain-confs/*.subdomain.conf;
```

Comment:
Should it be just `*.conf`?

Reply:
Should be resolved in daf99713a97f809748a67d6c3ec40ed1e1fcfe24